.mage/docs.go

```diff
@@ -17,13 +17,30 @@ package ttnmage
 import (
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
+	"io/ioutil"
+	"os"
+	"text/template"
 )
 
+type HugoConfig struct {
+	CurrentVersion string
+}
+
 // Docs namespace
 type Docs mg.Namespace
 
-// Gen Generates static website from the doc in doc/public/$version
+// Gen Generates static website from the doc in doc/public
 func (Docs) Gen() error {
+	mg.Deps(Version.getCurrent)
+	return sh.RunV(
+		"cp",
+		"-s", "./doc",
+		"--baseUrl", "https://thethingsnetwork.github.io/lorawan-stack/",
```

Comment:
Let's make this configurable per https://github.com/TheThingsIndustries/lorawan-stack/issues/1655